app/lib/scrapers/who_scraper.rb

```diff
@@ -55,6 +55,10 @@ def collect_data(disease_link)
   end
 
   def facts(disease_page)
-    disease_page.at('h3:contains("Key facts")').try(:next_element).try(:text)
+    parsed_lists = disease_page.at('h3:contains("Key facts")').try(:next_element)
+    if parsed_lists && (lists = parsed_lists.search("li")).present?
```

Comment:
if `false`, then the `result` will not be set. So it becomes `nil` in this case. It is fine when a serialized Array column with `nil` is set since it will return an empty arrary.